gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-all.zip
```

Comment:
You meant 4.8.1 or there is any incompatibility with the recent Gradle versions?

Reply:
It should be 4.8.1 thanks for pointing this out.